### ISSUE TITLE
Unable to view the dataset and loading the same dataset throws the error - ArrowInvalid: Exceeded maximum rows

### ISSUE THREAD

**Issue by Pavithree** [huggingface/datasets#4107]
## Dataset viewer issue - -ArrowInvalid: Exceeded maximum rows

**Link:** *https://huggingface.co/datasets/Pavithree/explainLikeImFive*

*This is the subset of original eli5 dataset  https://huggingface.co/datasets/vblagoje/lfqa. I just filtered the data samples which belongs to one particular subreddit thread. However, the dataset preview for train split  returns  the below mentioned error:
Status code:   400
Exception:     ArrowInvalid
Message:       Exceeded maximum rows
When I try to load the same dataset it returns ArrowInvalid: Exceeded maximum rows error*

Am I the one who added this dataset ? Yes

**Comment by severo** [comment]
Thanks for reporting. I'm looking at it

**Comment by lhoestq** [comment]
It seems that train.json is not a valid JSON Lines file: it has several JSON objects in the first line (the 915th character in the first line starts a new object, and there's no "\n")

You need to have one JSON object per line

**Comment by albertvillanova** [comment]
I'm closing this issue.

@Pavithree, please, feel free to re-open it if fixing the JSON file does not solve it.

**Comment by Pavithree** [comment]
Thank you! that fixes the issue.
